server/models/List.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require('mongoose')
+
+const ListSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  image: {
+    type: String,
+  },
+  products: [
+    { type: mongoose.Schema.Types.ObjectId, required: true, ref: 'Product' },
```

Comment:
Have you tested if a list can have no products in it when this required is set to true? Just be sure to confirm since empty lists should be possible

Reply:
Your right, since the list is initially created with no products, it shouldn't be required. I will make that change.